### ISSUE TITLE
Restoring all posts of an unblocked/restored user

### ISSUE THREAD

**Issue by Michael1337** [consul/consul#4019]
### Current Behavior
When an administrator unblocks/restores a user, they have to restore all comments of that user manually.

### Expected Behavior
When an administrator unblocks/restores a user, they should be asked if they also want to restore all posts by that user in one click.

### Steps to reproduce
Block a user that made at least one comment. Restore that user. Check moderated comments.

**Comment by Michael1337** [comment]
Just found this issue: https://github.com/consul/consul/issues/3551
Is this still worked on?

**Comment by javierm** [comment]
Hi, @Michael1337 :smile:. Nobody is currently working on it. Feel free to have a go at it if you'd like to :wink:.

Closing in favor of #3551.
